addons/website_sale/views/snippets/snippets.xml

```diff
@@ -24,23 +24,31 @@
             data-selector="#products_grid .oe_product"
             data-no-check="true">
             <div class="o_wsale_soptions_menu_sizes">
-                <we-title>Size</we-title>
-                <table>
-                    <tr>
-                        <td/><td/><td/><td/>
-                    </tr>
-                    <tr>
-                        <td/><td/><td/><td/>
-                    </tr>
-                    <tr>
-                        <td/><td/><td/><td/>
-                    </tr>
-                    <tr>
-                        <td/><td/><td/><td/>
-                    </tr>
-                </table>
+                <we-row string="Size" class="o_we_row_custom">
+                    <table>
+                        <tr>
+                            <td/><td/><td/><td/>
+                        </tr>
+                        <tr>
+                            <td/><td/><td/><td/>
+                        </tr>
+                        <tr>
+                            <td/><td/><td/><td/>
+                        </tr>
+                        <tr>
+                            <td/><td/><td/><td/>
+                        </tr>
+                    </table>
+                </we-row>
             </div>
 
+            <we-row string="Re-order" data-no-preview="true">
+                <we-button title="Push to top" data-change-sequence="top"><i class="fa fa-fw fa-angle-double-left"/></we-button>
+                <we-button title="Push up" data-change-sequence="up"><i class="fa fa-fw fa-angle-left"/></we-button>
+                <we-button title="Push down" data-change-sequence="down"><i class="fa fa-fw fa-angle-right"/></we-button>
+                <we-button title="Push to bottom" data-change-sequence="bottom"><i class="fa fa-fw fa-angle-double-right"/></we-button>
```

Comment:
Can you remove the `<i/>` tags and add the fa classes on the button directly?
It is supposed to be that way but right now is visually glitched.
But it will be fixed with https://github.com/odoo/odoo/pull/76313/files#r715622312
Note that you could use `o_we_button_icon` right now but.. better to wait for the final implementation of the automatic button icon.
cc @qsm-odoo